.env

```diff
@@ -4,18 +4,18 @@ MONGO_EXPRESS_PORT=8081
 REDIS_PORT=6379
 REDIS_INSIGHT_PORT=8001
 
-NODO_MOCK_COMMIT_SHA=master
+NODO_MOCK_COMMIT_SHA=fix-close-payment-request
 NODO_MOCK_PORT=3002
 
-ECOMMERCE_TRANSACTIONS_COMMIT_SHA=main
+ECOMMERCE_TRANSACTIONS_COMMIT_SHA=CHK-1258-user-cancel-api
 ECOMMERCE_TRANSACTIONS_PORT=8080
 
 ECOMMERCE_PAYMENT_METHODS_COMMIT_SHA=main
 ECOMMERCE_PAYMENT_METHODS_PORT=8083
 
 AFM_PORT=8085
 
-ECOMMERCE_SCHEDULER_COMMIT_SHA=main
+ECOMMERCE_SCHEDULER_COMMIT_SHA=CHK-1259-add-close-queue
```

Comment:
Are these PR branches needed? If so add a dependency on the relevant PR on this PR description

Reply:
All these branches will be restored to main before the merge